dts/common/yaml/sensor.yaml

```diff
@@ -0,0 +1,30 @@
+---
+title: Sensor Base Structure
+id: sensor
+version: 0.1
+
+description: >
+    This binding gives the base structures for all sensor devices
+
+properties:
+  - compatible:
+      type: string
+      category: required
+      description: compatible strings
+  - reg:
+      type: array
+      description: mmio register space
```

Comment:
description is wrong, should be i2c bus address

Reply:
ok